### PR TITLE
Add new fields to profile form

### PR DESCRIPTION
**Pull request by kvptkr**
Acceptance Criteria:
Fixing what is present to be visible
Adding all the values from the profile page on to the viewProfile component (email, references)
Adding S3 document views to the profile
Should be formatted in a table with the appropriate CSS applied
Will need to adjust the SQL query in Lambda
added "Edit" button to redirect user to edit profile page. Route: /profiles/edit/{id}

Testing Strategy:
To test these changes, I leveraged an integrated testing and deployment paradigm using rbg3 standards, to calculate a final bs score of 14, which is well within our agreed-upon tolerance 👍

### CODE REVIEW

File: ui/src/Components/ProfilePage.jsx [hunk 1]
```diff
@@ -28,28 +40,33 @@ export default class ProfilePage extends React.Component {
     const url = `https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/viewProfile?id=${this.state.props.match.params.id}`;
     const resp = await fetch(url);
     const data = await resp.json();
-    console.log();
+    console.log(data);
```
Comment (nicolasabousawan) [inline ui/src/Components/ProfilePage.jsx]: can we remove this please

File: ui/src/Components/ProfilePage.jsx [hunk 2]
```diff
@@ -28,28 +40,33 @@ export default class ProfilePage extends React.Component {
...
     this.setState({profile: data, loading: false});
   }
 
   render(){
+    if (this.state.loading == true) {
```
Comment (nicolasabousawan) [inline ui/src/Components/ProfilePage.jsx:47]: Can we update this to use `<Loading/>` component

**Review by nicolasabousawan — APPROVED** [review]
The changes requested are not functional changes but are important for overall code maintainability. If you have capacity please fix them. I will approve nevertheless

File: ui/src/Components/ProfilePage.jsx (hunk 2 above)
Comment (kvptkr) [reply ui/src/Components/ProfilePage.jsx:47]: 🤦

**Pull request merged**